Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Build Stage
+ARG PYTHON_VERSION=3.8
+FROM python:${PYTHON_VERSION} AS base
+WORKDIR /app
+COPY . . 
+
+# Run it babyyy
+FROM python:${PYTHON_VERSION}
```

Comment:
If you start to use multi-stage build,  keep in track/ You forget to add AS to this part

Reply:
@Alkaponees For some reason my build keeps failing using _AS base_ in a "Run" Stage. Can you please elaborate on this? I was trying to use _slim_ version but it failed as well.